src/sordid-dragon.jquery.coffee

```diff
@@ -135,31 +137,37 @@ do ($=jQuery) ->
         # Firefox won't trigger "drag" events without this.
         e.originalEvent.dataTransfer?.setData "text", ""
         $placeholder = $child.clone()
+        isDragging = true
 
       $handle.on "touchmove.sordidDragon drag.sordidDragon", (e) ->
-        showPlaceholder $child
-        if isTouch(e)
-          pageY = e.originalEvent.targetTouches[0].pageY
-          showGhost pageY
-
-          # On touch devices, we don't have dragenter events, so we'll update
-          # the position of the element being dragged here instead.
-          newPosition = currentPosition(pageY)
-          if newPosition?
-            moveChild $parent.children("#{options.childSelector || ""}:visible").eq(newPosition)
-          e.preventDefault()
+        if isDragging
```

Comment:
I'd rather just start the function with `return unless isDragging` than have to nest the rest of the function under the `if`.

If you are sure that we need `isDragging` that is fine. But it would be nice to have a comment explaining why. In theory these events should never be called unless we are dragging.